webpack/localization-plugin-5/src/test/LocalizedAsyncDynamic.test.ts

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+jest.disableAutomock();
+import { resolve } from 'path';
+import { promisify } from 'util';
+
+import webpack, { Compiler, Stats } from 'webpack';
+import { Volume } from 'memfs/lib/volume';
+
+import { LocalizationPlugin } from '../LocalizationPlugin';
+import type { ILocalizationPluginOptions, ILocalizationStats } from '../interfaces';
+import { MemFSPlugin } from './MemFSPlugin';
+
+jest.setTimeout(1e9);
+
+async function testLocalizedAsyncDynamicInner(minimize: boolean): Promise<void> {
+  const memoryFileSystem: Volume = new Volume();
+  memoryFileSystem.fromJSON(
+    {
+      '/a/package.json': '{ "name": "a", "sideEffects": ["entry.js", "async.js"] }',
+      '/a/async.js': `import strings1 from './strings1.resjson'; import strings2 from './strings2.resjson'; console.log(strings1.test, strings2.another);`,
+      '/a/entry.js': `import(/* webpackChunkName: 'async' */ './async');`,
+      '/a/strings1.resjson': `{"test":"blah","_test.comment":"A string"}`,
+      '/a/strings2.resjson': `{"another":"something else","_another.comment":"Another string"}`
+    },
+    '/'
+  );
+
+  let localizationStats: ILocalizationStats | undefined;
+  function statsCallback(stats: ILocalizationStats): void {
+    localizationStats = stats;
+  }
+
+  const resJsonLoader: string = resolve(__dirname, '../loaders/resjson-loader.ts');
+  const options: ILocalizationPluginOptions = {
+    localizedData: {
+      defaultLocale: {
+        localeName: 'en-us'
+      },
+      translatedStrings: {
+        foo: {
+          '/a/strings1.resjson': {
+            test: 'baz'
+          },
+          '/a/strings2.resjson': {
+            another: 'dead'
+          }
+        }
+      }
+    },
+    runtimeLocaleExpression: 'self.__locale',
+    localizationStats: {
+      callback: statsCallback
+    }
+  };
+
+  const localizationPlugin: LocalizationPlugin = new LocalizationPlugin(options);
+
+  const compiler: Compiler = webpack({
+    entry: {
+      main: '/a/entry.js'
+    },
+    output: {
+      path: '/release',
+      filename: '[name]-[locale].js',
+      chunkFilename: 'chunks/[name]-[locale].js'
+    },
+    module: {
+      rules: [
+        {
+          test: /\.resjson$/,
+          use: {
+            loader: resJsonLoader
```

Comment:
It'd be useful for the plugin's API to export the loader paths.

Reply:
Normally one would just write `@rushstack/webpack5-localization-plugin/lib/loaders/resjson-loader` or similar, but the unit tests have to do bizarre things due to the way Heft Jest works. You'll note that `resJsonLoader` is resolving to a `.ts` file.

Reply:
The paths are documented.

Reply:
Yeah I know in this case you need to reference the `.ts` file, but `@rushstack/webpack5-localization-plugin/lib/loaders/resjson-loader` is awkward and not very discoverable. It'd be more ergonomic if you could just say `LocalizationPlugin.resjsonLoaderPath`.